app/middlewares/errors.js

```diff
@@ -7,6 +7,7 @@ const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
   [errors.DEFAULT_ERROR]: 500,
   [errors.AUTHENTICATION_ERROR]: 401,
+  [errors.AUTHORIZATION_ERROR]: 403,
```

Comment:
I don't think 403 is the appropriate code for this, you should use `401` as well check this, related to `403`:
```The request was a legal request, but the server is refusing to respond to it. Unlike a 401 Unauthorized response, authenticating will make no difference.```